operator/v1/types_ingress.go

```diff
@@ -1037,6 +1039,18 @@ type SyslogLoggingDestinationParameters struct {
 // ContainerLoggingDestinationParameters describes parameters for the Container
 // logging destination type.
 type ContainerLoggingDestinationParameters struct {
+	// maxLength is the maximum length of the log message.
+	//
+	// Valid values are integers in the range 480 to 8192, inclusive.
+	//
+	// When omitted, the default value is 1024.
+	//
+	// +kubebuilder:validation:Optional
+	// +kubebuilder:validation:Maximum=8192
+	// +kubebuilder:validation:Minimum=480
+	// +kubebuilder:default=1024
```

Comment:
On the other hand, there are two default tags that you need, one for openapi and one for the crd. You should see the default in `openapi.json` once you add the second version
```suggestion
	// +kubebuilder:default=1024
	// +default:=1024
```